applications/vessel-history/src/routes/routes.ts

```diff
@@ -25,10 +26,17 @@ const thunk = async (
   getState: StateGetter<AppState>
 ) => null
 
+const homeThunk = async (
+  dispatch: Dispatch<AppActions | NavigationAction>,
+  getState: StateGetter<AppState>
+) => {
+  searchThunk(dispatch, getState)
+}
+
 const routesMap: RoutesMap = {
   [HOME]: {
     path: '/',
-    thunk,
+    thunk: homeThunk,
```

Comment:
@j8seangel @nerik What do you think about invoking thunks on routes? Yesterday we talked briefly about this with @fertandil87 because we did not find this pattern being applied in other repos regularly (fishing-map, carrier-portal) so we wonder how we want to move forward from now on on this? Thoughts, ideas, corrections are all welcome.